src/styles/styles.js

```diff
@@ -804,7 +804,7 @@ const styles = {
     },
     picker: (disabled = false, error = false, focused = false) => ({
         iconContainer: {
-            top: variables.pickerIconTop,
+            top: Math.round(variables.inputHeight * 0.5) - 10,
```

Comment:
This is weird, care to explain this?

Reply:
TO vertically center a child in its parent we need padding top = half parent height - half child height, in our case half input height = 26 and half icon height = 10 so we need padding top = 16, but because the input height increases when the device font size increases so instead of using a hard coded value we can make it dynamic.

Reply:
This calculation, itself seems off can we move the computation variables at least?

Reply:
Do you mean keeping `top: variables.pickerIconTop` in `styles.js` and in variables.js file using this `pickerIconTop: (getValueUsingPixelRatio(52, 72) * 0.5) - 10`?